package.json

```diff
@@ -146,7 +146,7 @@
     "rollup-plugin-uglify": "^3.0.0",
     "rollup-plugin-visualizer": "^0.1.5",
     "tslint": "^4.3.1",
-    "typescript": "^2.4.1"
+    "typescript": "2.9"
```

Comment:
this is a breaking change. a non breaking fix is here #1773

Reply:
That's better. Closing this